reconcile/openshift_saas_deploy.py

```diff
@@ -100,7 +102,13 @@ def slack_notify(
         message += f" triggered by _{trigger_integration}_"
     if in_progress and skip_successful_notifications:
         message += ". There will not be a notice for success."
-    slack.chat_post_message(message)
+    try:
+        slack.chat_post_message(message)
+    except SlackApiError as e:
```

Comment:
I suggest adding a small comment to explain why we are doing this i.e limitation of slack API to provide useful error message with the channel identifier and exposing user friendly log message to tenant.

Reply:
No problem at all.